README.meteoswiss.md

```diff
@@ -0,0 +1,74 @@
+FLEXPART for MeteoSwiss
+=======================
+
+FLEXPART for IFS (and GFS) adapted source code
+for operational MeteoSwiss production.
+
+
+Installation
+------------
+
+### Using the `spack` package
+
+Activate [spack-c2sm](https://github.com/C2SM/spack-c2sm/blob/main/README.md)
+and build flexpart with `spack`: 
+
+    spack dev-build -u build flexpart-ifs @develop %gcc
```

Comment:
The flexpart-ifs spack package still points to the flexpart-ifs repo, not this one. 
Or does the dev-build instead use the curent repo as source? https://github.com/C2SM/spack-c2sm/blob/main/repos/c2sm/packages/flexpart-ifs/package.py

Reply:
Good point. We should mention that this requires at least spack-c2sm v0.20.1.2, which will be released next week and I'm currently working on.